T_sensor:
  [1, 0, 0,
   0, 1, 0,
   0, 0, 1]
quat_uncertainty: 0.1
pose_uncertainty_m: 10
vel_uncertainty_mps: 10
gyro_bias_noise: 0.001 
acc_bias_noise: 0.01

quat_process_noise: 0.02
pos_process_noise_m: 0.5
vel_process_noise_mps: 0.2
gyro_process_noise_rps: 0.005
acc_process_noise_mps2: 0.001

cam_trans_means_m: 0.05
cam_pose_means: 0.5
imu_acc_means_mps2: 0.1

